content/blog/2018-06-22-new-editors.md

```diff
@@ -0,0 +1,49 @@
+
+<!---<img src="/img/blog-images/2018-06-22-new-editors/anna-krystalli.jpg" alt="Anna Krystalli" style="margin: 0px 20px; width: 250px;" align="left"> _quote from Anna_ --->
+
+<div style="overflow: hidden; width: 100%;">
+<img src="/img/blog-images/2018-06-22-new-editors/anna-krystalli.jpg" alt="Anna Krystalli" style="margin: 15px 25px 25px 0px; width: 250px;" align="left">
+</div>
+
+> quote from Anna
+
+**Anna Krystalli** is a Research Software Engineer at [University of Sheffield](http://rse.shef.ac.uk/) with a PhD in marine macroecology. She has helped organize [Reprohacks](http://rse.shef.ac.uk/blog/opencon_london/) where teams try to reproduce papers nominated by their authors from supplied code and data, and is one of Mozilla Foundation's "50 People Who Made the Internet a Better Place in 2016". Anna reviewed the [`codemetar`](https://github.com/ropensci/onboarding/issues/130) and [`rdflib`](https://github.com/ropensci/onboarding/issues/169) packages for rOpenSci. During her second package review, Anna started work on an experimental reviewing workflow package, [`pkgreviewr`](https://github.com/ropenscilabs/pkgreviewr).
+
+Anna on [GitHub](https://github.com/annakrystalli), [Twitter](https://twitter.com/annakrystalli), [website](http://annakrystalli.me/)
+
+<!---<img src="/img/blog-images/2018-06-22-new-editors/lincoln-mullen.jpg" alt="Lincoln Mullen" style="margin: 0px 20px; width: 250px;" align="left">  _quote from Lincoln_ --->
+
+<div style="overflow: hidden; width: 100%;">
+<img src="/img/blog-images/2018-06-22-new-editors/lincoln-mullen.jpg" alt="Lincoln Mullen" style="margin: 15px 25px 25px 0px; width: 250px;" align="left">
+</div>
+
+> quote from Lincoln
+
+**Lincoln Mullen** is an Assistant Professor at [George Mason University](https://historyarthistory.gmu.edu/) with a PhD in history. He is a historian of American religious history and the nineteenth-century United States, with expertise in computational methods for texts and maps. Lincoln maintains three rOpenSci packages [`tokenizers`](https://github.com/ropensci/tokenizers) for fast, consistent tokenization of natural language text, [`textreuse`](https://github.com/ropensci/textreuse) for detecting text reuse and document similarity, and [`USAboundaries`](https://github.com/ropensci/USAboundaries) for historical and contemporary state, county, and Congressional district boundaries, as well as zip code tabulation area centroids.
```

Comment:
I actually have three other packages that are a part of rOpenSci: historydata, gender, and internetarchive. If you don't want to mention those three, then you could say something like "maintains six rOpenSci packages, among them ..."

Reply:
@lmullen Do you have a relevant award or distinction I could mention? Or a related thing you're involved in/passionate about that would give some insight?

Reply:
It's generally not polite to bring up one's own awards. 😋

You could mention something like this:

> Mullen's historical research that uses R includes an article on legal history published in the *American Historical Review*, the leading journal in history; [America's Public Bible](http://americaspublicbible.org/), which won first-place in the National Endowment for the Humanities Chronicling America Data Challenge; and the NEH-funded project [Mapping Early American Elections](http://earlyamericanelections.org/).